web/src/features/cHaines/pages/CHainesPage.tsx

```diff
@@ -0,0 +1,736 @@
+    prediction_timestamp: string
+  ) => {
+    return (
+      model_abbreviation in model_run_predictions &&
+      model_run_timestamp in model_run_predictions[model_abbreviation] &&
+      prediction_timestamp in
+        model_run_predictions[model_abbreviation][model_run_timestamp]
+    )
+  }
+
+  useEffect(() => {
+    dispatch(fetchModelRuns(selectedDatetime))
+  }, []) // eslint-disable-line react-hooks/exhaustive-deps
+
+  useEffect(() => {
+    if (
+      selected_prediction_timestamp &&
+      selected_model_run_timestamp &&
+      model_runs.length > 0
+    ) {
+      loadModelPrediction(
+        selected_model_abbreviation,
+        selected_model_run_timestamp,
+        selected_prediction_timestamp
+      )
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [model_runs])
+
+  useEffect(() => {
+    if (selected_model_abbreviation in model_run_predictions) {
+      if (
+        selected_model_run_timestamp in model_run_predictions[selected_model_abbreviation]
+      ) {
+        if (
+          selected_prediction_timestamp in
+          model_run_predictions[selected_model_abbreviation][selected_model_run_timestamp]
+        ) {
+          showLayer(
+            selected_model_abbreviation,
+            selected_model_run_timestamp,
+            selected_prediction_timestamp
+          )
+        }
+      }
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [model_run_predictions])
+
+  useEffect(() => {
+    mapRef.current = L.map('map-with-selectable-wx-stations', {
+      center: [55, -123.6],
+      zoom: 5,
+      // scrollWheelZoom: false,
+      zoomAnimation: true
+      // layers: [topoLayer, stationOverlay]
+    })
+    L.control.scale().addTo(mapRef.current)
+
+    const baseLayer = tiledMapLayer({
+      //url:  'https://maps.gov.bc.ca/arcserver/rest/services/province/web_mercator_cache/MapServer'
+      url: 'https://maps.gov.bc.ca/arcserver/rest/services/Province/roads_wm/MapServer'
+    })
+    baseLayer.addTo(mapRef.current)
+
+    // Active weather stations layer.
+    L.tileLayer
+      .wms(
+        'https://openmaps.gov.bc.ca/geo/pub/WHSE_LAND_AND_NATURAL_RESOURCE.PROT_WEATHER_STATIONS_SP/ows?',
+        {
+          format: 'image/png',
+          layers: 'pub:WHSE_LAND_AND_NATURAL_RESOURCE.PROT_WEATHER_STATIONS_SP',
+          styles: 'BC_Wildfire_Active_Weather_Stations',
+          transparent: true,
+          minZoom: 0,
+          maxZoom: 18
+        }
+      )
+      .addTo(mapRef.current)
+
+    // Active weather station labels.
+    L.tileLayer
+      .wms(
+        'https://openmaps.gov.bc.ca/geo/pub/WHSE_LAND_AND_NATURAL_RESOURCE.PROT_WEATHER_STATIONS_SP/ows?',
+        {
+          format: 'image/png',
+          layers: 'pub:WHSE_LAND_AND_NATURAL_RESOURCE.PROT_WEATHER_STATIONS_SP',
+          styles: 'BC_Wildfire_Active_Weather_Stations_Labels',
+          transparent: true,
+          minZoom: 0,
+          maxZoom: 18
+        }
+      )
```

Comment:
do we need to add attributions for the sources of these layers?

Reply:
Leaflet automatically pulls in attribution and throws it in the bottom right corner.